src/safeposix/syscalls/net_calls.rs

```diff
@@ -36,29 +44,49 @@ impl Cage {
         return sockfd;
     }
 
+    //Find an available file descriptor index in the File Descriptor Table
+    //If available, insert the socket fd into the File Descriptor Table at the 
+    //available index
     fn _socket_inserter(&self, sockfd: FileDescriptor) -> i32 {
         let (fd, guardopt) = self.get_next_fd(None);
+        //In the case that no fd is available from the call to get_next_fd,
+        //fd is set to -ENFILE = -23 and the error is propagated forward
         if fd < 0 {
             return fd;
         }
         let fdoption = &mut *guardopt.unwrap();
-        let _insertval = fdoption.insert(sockfd);
+        //Insert the sockfd as the FileDescriptor inside fdoption
```

Comment:
The option doesn't say anything about whats inside it, it just is None or Some(). By inserting into it indicates that there is now a fd at that index